app/scripts/routes.js

```diff
@@ -16,6 +16,10 @@ define([], function() {
                 templateUrl: 'views/students/index.html',
                 controller: 'StudentsIndexCtrl',
                 relativePath: '/students'
+            },
+            '/students/:docket/update': {
```

Comment:
It's just that we've agreed to use [Rail's endpoints](http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) as a starting point.

I've told you to change it since there the edit's view path is `/photos/:id/edit`.
It's just fu**ing convention. We should play 'stone, paper or scissors' to define it :stuck_out_tongue_closed_eyes: 
What do you say?



Also, I have to admit that I've made a mistake. Note that the update path is `/photos/:id` with a PATCH/PUT HTTP verb (there is no `/update` at the end of the URL).
I've noticed that in the documentation we have used a POST verb. @gibarsin would you change it or leave it like that?